mobile/library/common/jni/jni_interface.cc

```diff
@@ -120,10 +121,24 @@ extern "C" JNIEXPORT jlong JNICALL Java_io_envoyproxy_envoymobile_engine_JniLibr
 }
 
 extern "C" JNIEXPORT jint JNICALL Java_io_envoyproxy_envoymobile_engine_JniLibrary_runEngine(
-    JNIEnv* env, jclass, jlong engine, jstring config, jstring log_level) {
+    JNIEnv* env, jclass, jlong engine, jstring config, jlong native_bootstrap, jstring log_level) {
```

Comment:
I know we don't seem to have comments on the JNICALL functions in this file, but I wonder if we should start. In particular, it would be good to comment on what "native_bootstrap" is (it's an optional pointer to a Bootstrap proto). It would also be good to explain what happens if it is null?

Also, I think @jpsim got rid of the "native_" prefix from fields in his recent PR. Do we have a convention for when that prefix is appropriate?

Reply:
documented in the JNILibrary butI forgot to update.
Also commented here and referenced that.

Reply:
I don't have a strong preference on the naming here, as long as we stay consistent with the Java interface to make it easier to produce the correct matching JNI.